src/main/java/com/foo/gosucatcher/domain/member/application/MemberEmailAuthService.java

```diff
@@ -0,0 +1,118 @@
+package com.foo.gosucatcher.domain.member.application;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.foo.gosucatcher.domain.member.application.dto.request.MemberEmailAuthRequest;
+import com.foo.gosucatcher.domain.member.application.dto.response.MemberEmailAuthResponse;
+import com.foo.gosucatcher.domain.member.application.dto.response.MemberEmailSendResponse;
+import com.foo.gosucatcher.domain.member.domain.MemberRepository;
+import com.foo.gosucatcher.global.error.ErrorCode;
+import com.foo.gosucatcher.global.error.exception.BusinessException;
+import com.foo.gosucatcher.global.error.exception.InvalidValueException;
+import com.foo.gosucatcher.global.util.EmailRedisTemplateUtils;
+import com.foo.gosucatcher.global.util.RandomNumberUtils;
+
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class MemberEmailAuthService {
+
+	private final EmailRedisTemplateUtils emailRedisTemplateUtils;
+	private final JavaMailSender javaMailSender;
+	private final MemberRepository memberRepository;
+
+	@Value("${auth.email.senderEmail}")
+	private String SENDER_EMAIL;
+	@Value("${auth.time.expiration}")
+	private Long EXPIRATION_TIME;
```

Comment:
이거 왜 대문자+언더바인가요...?
주입받는 변수들은 그렇게 하나요? 

```suggestion
	@Value("${auth.email.senderEmail}")
	private String senderEmail;
	
	@Value("${auth.time.expiration}")
	private Long expirationTime;
```

Reply:
주입 받는 변수들은 상수임에도 불구하고 static을 붙이지 못합니다..! 어째튼 개념상으로 상수니까...? 대문자로 했습니다!